handbook/talent/types_of_interviews.md

```diff
@@ -327,6 +327,12 @@ In addition to the above interviews, some departments have department-specific i
   - Duration: 60-minutes.
   - Details: during this stage in the interview process, we will get an understanding of the technical depth of your experience and how you would contribute to our engineering culture. We will ask you about your past work and accomplishments in depth, how you worked wiht others, decisions you made, and what you'd do differently today.
 
+- Architectural interview
+
+  - Interviewers(s): two members of the engineering department
+  - Duration: 45-minutes
```

Comment:
```suggestion
  - Duration: 45 minutes
```

I think hyphen only used when make it a adjective, not as a noun. cc @quinnkeast